airflow/providers/common/sql/hooks/sql.py

```diff
@@ -17,8 +17,9 @@
 import warnings
 from contextlib import closing
 from datetime import datetime
-from typing import Any, Optional
+from typing import TYPE_CHECKING, Any, Callable, Iterable, List, Mapping, Optional, Tuple, Union
 
+import sqlparse
```

Comment:
I belive this means we should add "sqlparse" to common.sql dependencies in provider. yaml

Reply:
Ah... I was confused by dependencies.json file. This file should have been deleted already - it was likely added at some rebase by mistake :(

Reply:
https://github.com/apache/airflow/pull/25108 removes the file